Week3 version 1

MingLin1995 opened the pull request:
林右銘
http://54.184.246.26:3000/
http://54.184.246.26:3000/attraction/10

Commit:
Merge branch 'develop'

Commit:
會員登入/註冊視窗

Commit:
Merge branch 'feature會員註冊登入視窗' into develop

Commit:
調整頁尾貼齊視窗底部

Commit:
更改取得imgURL邏輯，透過group_concat，會因為字數限制，導致取得的圖片網址被截斷，用id=11測試

Commit:
調整body底部，避免內文被頁尾遮蔽

Commit:
feat：頁尾修改
 因應新需求做調整：
1.修正內容不夠佈滿頁面時，footer沒有固定在底部的部分。
調整項目：
1.index.html
2.index.css

Commit:
feat：新增回首頁功能
 因應新需求做調整：
1.點擊台北一日遊，可以返回首頁
調整項目：
1.index.html
2.index.css

Commit:
feat：轉跳景點頁面、樣式微調
 因應新需求做調整：
1.點擊景點畫面，轉跳景點頁面
2.滑鼠移入圖片特效
調整項目：
1.index.js
2.index.css

Commit:
feat：景點頁面
 因應新需求做調整：
1.新增景點頁面
調整項目：
1.attraction.html
2.attraction.css
3.attraction.js
4.各種圖片

Commit:
Merge branch 'feature/景點頁面' into develop

Commit:
feat：修改樣式
 因應新需求做調整：
1.修改按鈕字體
2.修改RWD圖片高度
調整項目：
1.attraction.css

Commit:
Merge branch 'feature/景點頁面' into develop

cwpeng reviewed (changes requested):
1. There should be no space between navigator and attraction image in mobile devices.
2. Separator line on the bottom of navigator should be 1px.

Commit:
feat：樣式調整
 因應新需求做調整：
1.調整navigator與attraction之間的間隔
2.調整navigator底部Separator line的大小
調整項目：
1.attraction.css

Commit:
Merge branch 'feature/景點頁面' into develop

cwpeng reviewed (approved):
OK. Done.
